src/test/java/uk/gov/hmcts/darts/audio/component/impl/AddAudioRequestMapperImplTest.java

```diff
@@ -31,7 +31,7 @@ void setUp() {
     }
 
     @Test
-    void testMapToMedia() {
+    void givenCourtroomExistsOrCreated_whenMapCourtroomAudioMetadataToMediaIsRequested_thenMetadataIsSuccessfullyMapped() {
```

Comment:
This could get tedious when writing lots of unit tests. I tend to think of these as `shouldDoSomething`.

Reply:
true, for some test types it takes a bit of diligence and effort to come up with a good given/when/then name